build/Targets/Settings.props

```diff
@@ -49,6 +49,11 @@
     <VisualStudioCodename>Dev$(VisualStudioReferenceMajorVersion)</VisualStudioCodename>
     <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
 
+    <!-- When building WPF / VSIX projects MSBuild will create a temporary project with an extension of
+         tmp_proj.  In that case the SDK is unable to determine the target language and cannot pick
+         the correct import.  Need to set it explicitly here. -->
+    <LanguageTargets Condition="'$(ProjectLanguage)' != ''">$(MSBuildToolsPath)\Microsoft.$(ProjectLanguage).targets</LanguageTargets>
```

Comment:
>ProjectLanguage [](start = 35, length = 15)

Instead of setting a property manually, can we check the extension of the first file in Compile items?